### ISSUE TITLE
(b/69928690): support mixed precision in the XLA batch normalization 

### ISSUE THREAD

**Issue by piascikj** [imdone/tensorflow#336]
operators. For now, cast everything to the statistics type (which
may be more precise than the input type).

----
*opened via [imdone.io](https://imdone.io) from a code comment on [f283173062f3ff9b6f69e8fc8a77421dcfdaa8f2](https://github.com/imdone/tensorflow/commit/f283173062f3ff9b6f69e8fc8a77421dcfdaa8f2) by Peter Hawkins phawkins@google.com*

----

https://github.com/imdone/tensorflow/blob/8cb6e535f6ea14b380aa86d425169adad682cb8c/tensorflow/compiler/tf2xla/kernels/batch_norm_op.cc#L125-L132
